dojo/tools/pip_audit/parser.py

```diff
@@ -0,0 +1,74 @@
+import json
+
+from dojo.models import Finding
+
+
+class PipAuditParser:
+
+    def get_scan_types(self):
+        return ["pip-audit Scan"]
+
+    def get_label_for_scan_types(self, scan_type):
+        return "pip-audit Scan"
+
+    def get_description_for_scan_types(self, scan_type):
+        return "Import pip-audit JSON scan report."
+
+    def requires_file(self, scan_type):
+        return True
+
+    def get_findings(self, scan_file, test):
+
+        scan_data = scan_file.read()
+        try:
+            data = json.loads(str(scan_data, 'utf-8'))
+        except:
+            data = json.loads(scan_data)
+
+        findings = list()
+        for item in data:
+            vulnerabilities = item.get('vulns', [])
+            if vulnerabilities:
```

Comment:
We should generate an `Info` finding with only the data for the component.
It could be useful if there is no vulnerabilities to have at least the data for components.

Reply:
I wouldn't like to use DefectDojo to store a BOM (bill of material), we should stick to manage vulnerabilities.